### DISCUSSION TITLE
Are table indexes stored in a saved tree sequence?

### DISCUSSION THREAD

**Discussion by hyanwong** [tskit-dev/tskit#3074] [Q&A] [answered]
When you call `ts.dump()`, does the resulting file contain the table indexes, or do they have to be created each time the tree sequence is reloaded from disk. I assume the former. If so, can I add that information to https://tskit.dev/tskit/docs/latest/data-model.html#table-indexes?

**Comment by jeromekelleher** [comment, marked as answer]
Yep, you can't create a TreeSequence object without indexes, and these are stored to file if present in a TableCollection (code [here](https://github.com/tskit-dev/tskit/blob/14f8ed2dc8d648cd43975137a6794c7e0d5c46fe/c/tskit/tables.c#L11632)
